### PR TITLE
Restructure README

### PR DESCRIPTION
**Pull request by trungdlp-wolffun**
### What is the purpose of this pull request?
I want to restructure README to focus core value of this repository
### When do you want to merge?
A about 1 a week

Could you please review/proofread my work?
[Visit this page](https://github.com/trungdlp-wolffun/ezyfox-server/tree/chore/update_readme)

### CODE REVIEW

File: README.md
Comment (tvd12) [inline README.md:54]: - An EzyFox Server will contain multiple zones.
- Each Zone will contain a zone's user manager, multiple apps, and multiple plugins.
- An App will contain an app's user manager.
- A Plugin will only handle events and client's requests.

**Review by tvd12 — CHANGES_REQUESTED** [review]
@trungdlp-wolffun PLTAL

**Comment by tvd12** [comment]
@vu-luong do you like this readme style? Take a look [here](https://github.com/trungdlp-wolffun/ezyfox-server/tree/chore/update_readme) for preview

File: README.md
Comment (trungdlp-wolffun) [reply README.md:54]: Okay, I will update soon.
@tvd12 please update on your website at here: https://youngmonkeys.org/ezyfox-server/guides/ezyfox-server-architecture

Comment (tvd12) [reply README.md:54]: updated, thank you

**Comment by trungdlp-wolffun** [comment]
@tvd12 I think we need add example for Maven and Gradle user to install EzyFox, like this:

image

**Comment by tvd12** [comment]
> @tvd12 I think we need add example for Maven and Gradle user to install EzyFox, like this:
> 
> image

Let me think, but I don't prefer it, because ezyfox server is difference from netty, it has multi components and have some way to create a project and start coding, one of way you can refer here: https://youngmonkeys.org/ezyfox-server/guides/create-ezyfox-server-project

File: README.md
Comment (vu-luong) [inline README.md:56]:
```suggestion
See more detail about the sever architecture [here](https://youngmonkeys.org/ezyfox-server/guides/ezyfox-server-architecture)
```

**Review by vu-luong — CHANGES_REQUESTED** [review]

File: README.md
Comment (vu-luong) [reply README.md:54]:
```suggestion
- An EzyFox server contains multiple zones
- A zone contains its user manager, multiple apps and multiple plugins
- An app contains its user manager
- A plugin only handles events and clients' requests
```

Comment (tvd12) [reply README.md:54]: @trungdlp-wolffun could you update by Vu suggestion? I see it's more suitable

**Comment by TrungKenbi** [comment]
@vu-luong thank for you suggestion, I updated.
cc @tvd12

File: README.md
Comment (TrungKenbi) [reply README.md]: @tvd12 I updated.

File: README.md
Comment (vu-luong) [inline README.md:37]:
```suggestion
[Let's see how we can get started in just 5 minutes!](https://youngmonkeys.org/get-started/)
```

File: README.md
Comment (vu-luong) [inline README.md:69]:
```suggestion
1. [EzyChat](https://youtube.com/playlist?list=PLlZavoxtKE1IfKY7ohkLLyv6YkHMkvH6G): A simple real-time chat application
```

File: README.md
Comment (vu-luong) [inline README.md:104]: > we'd like to keep track of our users

This sounds quite privacy-sensitive to me. Maybe consider rewording it to:
```suggestion
As the EzyFox community grows, we'd love to see how our open-source framework can help you and your organization solve real-world problems. Feel free to send a PR with your project name/organization name if you are using EzyFox.
```

**Review by tvd12 — APPROVED** [review]
LGTM

**Review by vu-luong — CHANGES_REQUESTED** [review]
@TrungKenbi PLTAL

**Comment by TrungKenbi** [comment]
@vu-luong I updated. We should add demo code section?

**Comment by tvd12** [comment]
@trungdlp-wolffun I don't want to justify center the README content because I think people look form left to right, so I want to justify left, how do you think?

**Comment by TrungKenbi** [comment]
@tvd12 Hmm, I just center only image content and people love at the first sight when see great image.

**Comment by tvd12** [comment]
@vu-luong I will merge it after you approve

File: README.md
Comment (vu-luong) [inline README.md:41]:
```suggestion
- [Official EzyFox Server documentation](https://youngmonkeys.org/projects/ezyfox-server)
```

File: README.md
Comment (vu-luong) [inline README.md:42]:
```suggestion
- [Issues](https://github.com/youngmonkeys/ezyfox-server/issues)
```

Comment (vu-luong) [reply README.md:42]: @TrungKenbi A couple of final suggestions, thanks

Comment (trungdlp-wolffun) [reply README.md]: Finally, I updated it all. Many thanks for your suggestion @vu-luong

**Review by vu-luong — APPROVED** [review]
Thanks for the PR, LGTM!

**Pull request merged**
